context/app/static/js/components/savedLists/EditSavedStatusDialog/EditSavedStatusDialog.jsx

```diff
@@ -46,22 +47,30 @@ function EditSavedStatusDialog({ dialogIsOpen, setDialogIsOpen, uuid }) {
     setDialogIsOpen(false);
   }
 
+  const savedListsExist = Object.keys(savedLists).length > 0;
+
   return (
     <DialogModal
       title="Edit Saved Status"
       content={
-        <AddToList
-          selectedLists={selectedLists}
-          addToSelectedLists={addToSelectedLists}
-          removeFromSelectedLists={removeFromSelectedLists}
-        />
+        savedListsExist ? (
+          <AddToList
+            selectedLists={selectedLists}
+            addToSelectedLists={addToSelectedLists}
+            removeFromSelectedLists={removeFromSelectedLists}
+          />
+        ) : (
+          <Typography>
+            {`You don't have any lists. To assign this ${entity_type.toLowerCase()} to one or more lists, create a list in My Lists.`}
```

Comment:
Is it possible to make "My Lists" a link to the ".../my-lists" URL?

Reply:
<img width="831" alt="Screen Shot 2021-02-11 at 11 43 01 AM" src="https://user-images.githubusercontent.com/62477388/107668399-a243a580-6c5e-11eb-9d38-762033ff9012.png">